understanding/22/focus-not-obscured-minimum.html

```diff
@@ -91,7 +96,7 @@ <h3>Additional Techniques (Advisory)</h3>
          <h3>Failures</h3>
          <ol>
             <li>
-
+              It is essentially a bug (and a failure against this SC) if a component (for example, a hamburger menu) stays open on keyboard focus after keyboard focus is removed &ndash; when mouse-over behavior works as is typically expected (that is, the component automatically expands, but then collapses after the mouse cursor moves away from the component).
```

Comment:
I'm not sure why we're suddenly talking about mouse operation, and I am not agreeing that the 'expected' behaviour of mouse interaction is for menus to automatically expand on hover, if that is what is being inferred.

Reply:
I agree that we do not want to imply automatic expand on hover is expected behavior.

I also agree it would be better to avoid "mouse operation".